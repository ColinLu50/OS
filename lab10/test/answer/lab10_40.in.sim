7581
185
185
179
185
179
