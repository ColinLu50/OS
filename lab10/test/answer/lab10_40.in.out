FCFS
1 -> 106 -> 19 -> 31 -> 119 -> 139 -> 59 -> 102 -> 141 -> 85 -> 135 -> 6 -> 115 -> 95 -> 93 -> 153 -> 136 -> 8 -> 58 -> 108 -> 58 -> 57 -> 181 -> 59 -> 42 -> 25 -> 164 -> 59 -> 149 -> 36 -> 160 -> 115 -> 0 -> 12 -> 81 -> 183 -> 120 -> 128 -> 150 -> 60 -> 172 -> 34 -> 84 -> 74 -> 45 -> 179 -> 55 -> 0 -> 124 -> 160 -> 163 -> 125 -> 115 -> 6 -> 145 -> 138 -> 133 -> 64 -> 22 -> 59 -> 17 -> 12 -> 140 -> 65 -> 167 -> 150 -> 91 -> 173 -> 158 -> 139 -> 117 -> 9 -> 178 -> 156 -> 112 -> 129 -> 184 -> 51 -> 161 -> 45 -> 173 -> 128 -> 104 -> 182 -> 116 -> 135 -> 17 -> 0 -> 74 -> 108 -> 108 -> 136 -> 8 -> 149 -> 73 -> 149 -> 10 -> 77 -> 31 -> 70 -> 138 -> 154 -> 123 -> 156 -> 107 -> 47 -> 60 -> 123 -> 37 -> 81 -> 168 -> 11 -> 103 -> 160 -> 98 -> 68 -> 174
7581
SSTF
1 -> 0 -> 0 -> 0 -> 6 -> 6 -> 8 -> 8 -> 9 -> 10 -> 11 -> 12 -> 12 -> 17 -> 17 -> 19 -> 22 -> 25 -> 31 -> 31 -> 34 -> 36 -> 37 -> 42 -> 45 -> 45 -> 47 -> 51 -> 55 -> 57 -> 58 -> 58 -> 59 -> 59 -> 59 -> 59 -> 60 -> 60 -> 64 -> 65 -> 68 -> 70 -> 73 -> 74 -> 74 -> 77 -> 81 -> 81 -> 84 -> 85 -> 91 -> 93 -> 95 -> 98 -> 102 -> 103 -> 104 -> 106 -> 107 -> 108 -> 108 -> 108 -> 112 -> 115 -> 115 -> 115 -> 116 -> 117 -> 119 -> 120 -> 123 -> 123 -> 124 -> 125 -> 128 -> 128 -> 129 -> 133 -> 135 -> 135 -> 136 -> 136 -> 138 -> 138 -> 139 -> 139 -> 140 -> 141 -> 145 -> 149 -> 149 -> 149 -> 150 -> 150 -> 153 -> 154 -> 156 -> 156 -> 158 -> 160 -> 160 -> 160 -> 161 -> 163 -> 164 -> 167 -> 168 -> 172 -> 173 -> 173 -> 174 -> 178 -> 179 -> 181 -> 182 -> 183 -> 184
185
SCAN
1 -> 0 -> 0 -> 6 -> 8 -> 9 -> 10 -> 11 -> 12 -> 17 -> 19 -> 22 -> 25 -> 31 -> 34 -> 36 -> 37 -> 42 -> 45 -> 47 -> 51 -> 55 -> 57 -> 58 -> 59 -> 60 -> 64 -> 65 -> 68 -> 70 -> 73 -> 74 -> 77 -> 81 -> 84 -> 85 -> 91 -> 93 -> 95 -> 98 -> 102 -> 103 -> 104 -> 106 -> 107 -> 108 -> 112 -> 115 -> 116 -> 117 -> 119 -> 120 -> 123 -> 124 -> 125 -> 128 -> 129 -> 133 -> 135 -> 136 -> 138 -> 139 -> 140 -> 141 -> 145 -> 149 -> 150 -> 153 -> 154 -> 156 -> 158 -> 160 -> 161 -> 163 -> 164 -> 167 -> 168 -> 172 -> 173 -> 174 -> 178 -> 179 -> 181 -> 182 -> 183 -> 184
185
C-SCAN
1 -> 0 -> 0 -> 184 -> 184 -> 183 -> 182 -> 181 -> 179 -> 178 -> 174 -> 173 -> 172 -> 168 -> 167 -> 164 -> 163 -> 161 -> 160 -> 158 -> 156 -> 154 -> 153 -> 150 -> 149 -> 145 -> 141 -> 140 -> 139 -> 138 -> 136 -> 135 -> 133 -> 129 -> 128 -> 125 -> 124 -> 123 -> 120 -> 119 -> 117 -> 116 -> 115 -> 112 -> 108 -> 107 -> 106 -> 104 -> 103 -> 102 -> 98 -> 95 -> 93 -> 91 -> 85 -> 84 -> 81 -> 77 -> 74 -> 73 -> 70 -> 68 -> 65 -> 64 -> 60 -> 59 -> 58 -> 57 -> 55 -> 51 -> 47 -> 45 -> 42 -> 37 -> 36 -> 34 -> 31 -> 25 -> 22 -> 19 -> 17 -> 12 -> 11 -> 10 -> 9 -> 8 -> 6
179
LOOK
1 -> 0 -> 6 -> 8 -> 9 -> 10 -> 11 -> 12 -> 17 -> 19 -> 22 -> 25 -> 31 -> 34 -> 36 -> 37 -> 42 -> 45 -> 47 -> 51 -> 55 -> 57 -> 58 -> 59 -> 60 -> 64 -> 65 -> 68 -> 70 -> 73 -> 74 -> 77 -> 81 -> 84 -> 85 -> 91 -> 93 -> 95 -> 98 -> 102 -> 103 -> 104 -> 106 -> 107 -> 108 -> 112 -> 115 -> 116 -> 117 -> 119 -> 120 -> 123 -> 124 -> 125 -> 128 -> 129 -> 133 -> 135 -> 136 -> 138 -> 139 -> 140 -> 141 -> 145 -> 149 -> 150 -> 153 -> 154 -> 156 -> 158 -> 160 -> 161 -> 163 -> 164 -> 167 -> 168 -> 172 -> 173 -> 174 -> 178 -> 179 -> 181 -> 182 -> 183 -> 184
185
C-LOOK
1 -> 0 -> 184 -> 183 -> 182 -> 181 -> 179 -> 178 -> 174 -> 173 -> 172 -> 168 -> 167 -> 164 -> 163 -> 161 -> 160 -> 158 -> 156 -> 154 -> 153 -> 150 -> 149 -> 145 -> 141 -> 140 -> 139 -> 138 -> 136 -> 135 -> 133 -> 129 -> 128 -> 125 -> 124 -> 123 -> 120 -> 119 -> 117 -> 116 -> 115 -> 112 -> 108 -> 107 -> 106 -> 104 -> 103 -> 102 -> 98 -> 95 -> 93 -> 91 -> 85 -> 84 -> 81 -> 77 -> 74 -> 73 -> 70 -> 68 -> 65 -> 64 -> 60 -> 59 -> 58 -> 57 -> 55 -> 51 -> 47 -> 45 -> 42 -> 37 -> 36 -> 34 -> 31 -> 25 -> 22 -> 19 -> 17 -> 12 -> 11 -> 10 -> 9 -> 8 -> 6
179
